packages/shortest/src/ai/tools/index.ts

```diff
@@ -30,6 +30,42 @@ export const AITools = [
       required: ["action", "username", "password"],
     },
   },
+  {
+    name: "check_email",
+    description: "Fetch, decode and render received email in browser",
```

Comment:
Fetch / decode are implementation details. 

```suggestion
    description: "View received email in new browser tab",
```